### DISCUSSION TITLE
Mobile: Using an existing person name for an unmatched face doesn't prompt to merge

### DISCUSSION THREAD

**Discussion by eygraber** [immich-app/immich#7833] [Feature Request]
### The bug

If I add a name for an unmatched face in a photo, and use a name that already exists for a person, I don't get asked if I want to merge it into the existing person.

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

v1.98.0

### Version of Immich Mobile App

v1.98.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Tap Add name on a face in a photo on the mobile app
2. Add a name of an already existing person
```

```[tasklist]
### Tasks
```

**Comment by martabal** [comment]
That's a feature request ?

**Comment by eygraber** [comment]
I guess that's just how I expect the feature works 😬
